examples/simple/lint/L-CEIE1-bad.c

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2015 Runtime Verification, Inc. (RV-Match team). All Rights Reserved.
+
+#include <stdlib.h>
+#include <stdio.h>
+
+int main()
```

Comment:
I've added another whitespace here to adhere to a consistent format, described in one of the later comments.